src/agent/onefuzz-task/src/tasks/coverage/dotnet.rs

```diff
@@ -30,7 +30,7 @@ use crate::tasks::{
 };
 
 const MAX_COVERAGE_RECORDING_ATTEMPTS: usize = 2;
-const DEFAULT_TARGET_TIMEOUT: Duration = Duration::from_secs(5);
+const DEFAULT_TARGET_TIMEOUT: Duration = Duration::from_secs(120);
```

Comment:
FWIW, this is fine, but doesn't ever apply to the code path that is failing tasks. .NET coverage tasks wholly rely on the external `dotnet-coverage` tool.